static/home.js

```diff
@@ -90,3 +95,33 @@ function toggleModal(modal){
         $("#" + modal + "-arrow").removeClass("fa-angle-down");
     }
 };
+
+function setFac(facName){
+    if(facName == "" || facName == null) return;
+
+    if(facName == "mycourses"){
+        $("#programs").attr("src", "/catalog/finder/courses/" + facName + "/false");
+    } else { 
+        $("#programs").attr("src", "/catalog/finder/programs/" + facName + "/false");
+    }
+}
+
+function setProgram(programSlug){
+    if(programSlug == "" || programSlug == null) return;
+
+    $("#blocs").attr("src", "/catalog/finder/blocs/" + programSlug + "/false");
+}
+
+function setBloc(blocId){
+    if(blocId == "" || blocId == null) return;
+
+    $("#courses").attr("src", "/catalog/finder/courses/" + blocId + "/false");
+}
+
+function load_initial_state(){
+    var url = new URL(window.location);
+
+    setFac(url.searchParams.get("facs"));
+    setProgram(url.searchParams.get("programs"));
+    setBloc(url.searchParams.get("blocs"));
+}
```

Comment:
`\n` at the end of file? 👀

Reply:
This file will probably be dropped during the merge :upside_down_face:

Reply:
🙃